### PR TITLE
Change sortId to currentField in sort directive

### PR DESCRIPTION
**Pull request by karelhala**
In documentatin was type, where it said to use sortId to select element from which items should be filtered, however currentField is the option which shuld be used.

No changes in code, just in usage documentation.

### CODE REVIEW

**Comment by karelhala** [comment]
I am not sure whether dist folder should be pushed as well, but since its using bower I assume it's nice to keep changes in dist for upstream, right?

File: dist/angular-patternfly.js [hunk 1]
```diff
@@ -775,10 +775,12 @@ angular.module('patternfly.card').directive('pfCard', function () {
...
-            chart = c3.generate(chartData);
-            if (scope.getChartCallback) {
-              scope.getChartCallback(chart);
+            if (chartData) {
+              chartData.bindto = '#' + attrs.id;
+              chart = c3.generate(chartData);
+              if (scope.getChartCallback) {
+                scope.getChartCallback(chart);
+              }
             }
```
Comment (jeff-phillips-18) [inline dist/angular-patternfly.js]: Looks like an issue with a rebase here. This is removing the change from #226

**Comment by jeff-phillips-18** [comment]
Yes, these dist files do need to be included.

File: dist/angular-patternfly.js (hunk 1 above)
Comment (karelhala) [inline dist/angular-patternfly.js]: Yeah, I figured that #226 did not included dist folder. I'll keep it in.

**Comment by jeff-phillips-18** [comment]
LGTM 👍

**Comment by dtaylor113** [comment]
LGTM

**Pull request merged**